app/src/main/java/nu/yona/app/api/manager/impl/PasscodeManagerImpl.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2016 Stichting Yona Foundation
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package nu.yona.app.api.manager.impl;
+
+import nu.yona.app.YonaApplication;
+import nu.yona.app.api.manager.PasscodeManager;
+import nu.yona.app.utils.PreferenceConstant;
+
+/**
+ * Created by bhargavsuthar on 3/31/16.
+ */
+public class PasscodeManagerImpl implements PasscodeManager {
+
+    private int counter = 0;
+    private final int max_counter = 4;
```

Comment:
@bhargav1 can we define this in AppConstant, so we can get all setting related stuffs at same place?